cmd/rollkit/commands/run_node.go

```diff
@@ -128,6 +135,16 @@ func NewRunNodeCmd() *cobra.Command {
 				defer func() { srv.Stop(cmd.Context()) }()
 			}
 
+			// use mock grpc sequencer server by default
+			if !cmd.Flags().Lookup("rollkit.sequencer_address").Changed {
```

Comment:
Does the `.Changed` boolean get triggered if you set it to the default? 
If so, then we might what to updated this to explicitly check if the address is the default.